include/wx/uilocale.h

```diff
@@ -97,11 +108,11 @@ class WXDLLIMPEXP_BASE wxUILocale
     // Configure the UI to use the default user locale.
     static bool UseDefault();
 
-    // Use the locale corresponding to the given language.
+    // Use the locale corresponding to the given locale name.
```

Comment:
Even though it's an internal function, it would be nice to explain what is "name" in this context. I'm not 100% sure I'm understand it correctly yet, but maybe something like this:
```suggestion
    // Use the locale corresponding to the given POSIX locale, e.g. "de_DE.UTF-8".
```
?